packages/paste-design-tokens/tokens/themes/dark/global/box-shadow.yml

```diff
@@ -23,15 +23,21 @@ props:
   shadow-focus-inverse:
     value: "{!offset-0} {!offset-0} {!offset-0} {!offset-20} {!palette-gray-0-transparent-40}"
     comment: Shadow for focus ring on interactive elements on inverse backgrounds.
-  shadow-border-stronger:
-    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-60}"
+  shadow-border-weaker:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-20}"
     comment: Lighter shadow border for disabled inputs.
-  shadow-border-strong:
-    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-90}"
+  shadow-border-weak:
+    value: "{!offset-0} {!offset-0} {!offset-0} 1px {!palette-gray-30}"
```

Comment:
This is where the regression comes in for darkmode. The values have been copied across from default

Reply:
@SiTaggart good catch. I fixed that regression in the dark theme.

Reply:
Actually chatted with @serifluous about this one. The intended disabled input should be like this: https://share.getcloudapp.com/v1uYRjwk

So it looks like it was broken before.